core/src/test/java/org/mskcc/cbio/portal/scripts/TestIntegrationTest.java

```diff
@@ -176,7 +205,76 @@ public void testLoadStudyEs0() {
             }
             assertEquals(50, countGte2Lt3);
     		
+            //===== check cancer_type
+            List<DBCancerType> cancerTypes = apiService.getCancerTypes(Arrays.asList("brca-es0"));
+            assertEquals(1, cancerTypes.size());
+            assertEquals("Breast Invasive Carcinoma", cancerTypes.get(0).name);
+            
+            //===== check fusion data
+            //TODO - depends on fix for #1102 and #1314
+            
+            
+            //===== check gistic data
+            //servlet uses this query:
+            ArrayList<Gistic> gistics = DaoGistic.getAllGisticByCancerStudyId(cancerStudy.getInternalId());
+            assertEquals(14, gistics.size());
+            Gistic gisticChr10 = null, gisticChr20 = null;
+            for (Gistic gistic: gistics) {
+                if (gistic.getChromosome() == 20) {
+                    //assert not yet set:
+                    assertEquals(null, gisticChr20);
+                    gisticChr20 = gistic;
+                }
+                else if (gistic.getChromosome() == 10) {
+                    //assert not yet set:
+                    assertEquals(null, gisticChr10);
+                    gisticChr10 = gistic;
+                }
+            }
+            assertEquals(8, gisticChr10.getGenes_in_ROI().size());
+            assertEquals(1, gisticChr20.getGenes_in_ROI().size());
+            assertEquals("ZNF217", gisticChr20.getGenes_in_ROI().get(0).getHugoGeneSymbolAllCaps());
+            
+            //===== check methylation
+            geneticProfileStableIds = new ArrayList<String>();
+            geneticProfileStableIds.add("study_es_0_methylation_hm27");
+            hugoGeneSymbols = new ArrayList<String>(Arrays.asList("ATP2A1","SLMAP","HOXD3","PANX1","IMPA2","RHOC","TAF15","CCDC88B"));
+    		List<Serializable> methylationProfileData = apiService.getGeneticProfileData(geneticProfileStableIds, hugoGeneSymbols, null, null);
+    		//there is data for 8 genes x 311 samples:
+    		assertEquals(8*311, methylationProfileData.size());
+    		//simple check: there are 199 entries that have value between 0.5 and 0.6;
+    		int count0506Pos = 0;
+            for (Serializable profileEntry: methylationProfileData) {
+                String profileData = ((DBSimpleProfileData)profileEntry).profile_data;
+                double profileDataValue = Double.parseDouble(profileData);
+                if (profileDataValue >= 0.5 && profileDataValue < 0.6) {
```

Comment:
Same range used here (for methylation) as for CNA profiles. This is surprising, but I don't know these data values.